### DISCUSSION TITLE
docker start node error

### DISCUSSION THREAD

**Discussion by julycanfly** [MinaProtocol/mina#11437] [Q&A]
1.docker run  -d -p 9929:3087 --name mina minaprotocol/mina-rosetta:1.3.1-3e3abec-bionic
2.docker logs -f mina
3.print logs

========================= STARTING ROSETTA API on PORT 3087 ===========================
mina-rosetta: error while loading shared libraries: libjemalloc.so.1: cannot open shared object file: No such file or directory
========================= STARTING ARCHIVE NODE on PORT 3086 ===========================
mina-archive: error while loading shared libraries: libjemalloc.so.1: cannot open shared object file: No such file or directory
Removing daemon lockfile /data/.mina-config/.mina-lock
========================= STARTING DAEMON connected to MAINNET with GRAPQL on PORT 3085===========================
MINA Flags: --config-dir /data/.mina-config --peer-list-url https://storage.googleapis.com/seed-lists/mainnet_seeds.txt --external-port 10101 --rest-port 3085 -archive-address 127.0.0.1:3086 -insecure-rest-server --log-level Debug --log-json -config-file /genesis_ledgers/mainnet.json
mina: error while loading shared libraries: libjemalloc.so.1: cannot open shared object file: No such file or directory

========================= POPULATING MISSING BLOCKS ===========================
./docker-start.sh: line 99: kill: (181) - No such process
[FATAL] Mina daemon failed to start, exiting docker-start.sh
========================= CLEANING UP ===========================
Stopping mina daemon and waiting 3 seconds
mina: error while loading shared libraries: libjemalloc.so.1: cannot open shared object file: No such file or directory
Killing archive node
Killing mina daemon
Killing rosetta api
Stopping postgres cluster
mina-missing-blocks-auditor: error while loading shared libraries: libjemalloc.so.1: cannot open shared object file: No such file or directory
[BOOTSTRAP]

**Comment by shimkiv** [comment]
Hey @leezijie, not sure what is the question so I'm assuming complains are about errors experienced. 
Can you please tell if you need to run the `mina-rosetta` for reason or you missed an image?
